DEV_GUIDE.md

```diff
@@ -0,0 +1,152 @@
+# PlayerUI iOS Development Guide
+
+### Sections:
```

Comment:
probably worth to make Dependencies and Testings as separate sections as they are not only related to adding a new plugin